lighthouse-core/gather/gatherers/cache-contents.js

```diff
@@ -39,11 +39,10 @@ function getCacheContents() {
                 requests.push(...reqs.map(r => r.url));
               });
         })).then(_ => {
-          // __returnResults is magically inserted by driver.evaluateAsync
-          __returnResults(requests);
+          return requests;
         });
       }).catch(_ => {
-        __returnResults(undefined);
+        // Empty catch
```

Comment:
while you're changing this, maybe we should just drop the `catch` handler here? It isn't providing additional feedback and a rejected promise from `evaluateAsync` is caught in `afterPass` anyways